### PR TITLE
New doc: Add Overview part

### PR DESCRIPTION
**Pull request by daniceexi**

### CODE REVIEW

**Comment by daniceexi** [comment]
You can review the doc through my local web site. http://10.4.14.20/

File: docs/source/overview/index.rst [hunk 1]
```diff
@@ -1,14 +1,217 @@
...
 
-xCAT is known for exceptional scaling, wide variety of supported hardware, operating systems, and virtalization platforms, and complete day0 setup capabilities. 
+xCAT is known for exceptional scaling, wide variety of supported hardware, operating systems, and virtualization platforms. And complete day0 setup capabilities.
 
-.. toctree::
-   :maxdepth: 2
+xCAT Differentiators
+--------------------
+
+* xCAT Scales
```
Comment (zet809) [inline docs/source/overview/index.rst]: A separator(: or -) between title and content will make the doc much clear.

File: docs/source/overview/index.rst [hunk 2]
```diff
@@ -1,14 +1,217 @@
...
+
+   For a bigger cluster, you can use the automatic method to discover the target nodes. The discovered nodes will be defined to xCAT database. You can use ``lsdef`` to display them.
+
+   Refer to the doc: :doc:`xCAT discovery Guide <../guides/admin-guides/manage_clusters/ppc64le/discovery/index>` to learn how to discover and define compute nodes.
+
+#. Try to perform the hardware control against the target nodes
+
+   Now you have the node definition. Take a try to confirm the hardware control for defined nodes is working. e.g. ``rpower <node> stat``.
+
+   Refer to the doc: :doc:`Hardware Management <../guides/admin-guides/manage_clusters/ppc64le/management/index>` to learn how to perform the hardware discovery.
```
Comment (zet809) [inline docs/source/overview/index.rst]: ... hardware management.

**Comment by zet809** [comment]
Nice doc, I agree to merge it.

**Pull request merged**